Packages/CommonUI/Sources/CommonUI/View/TextField/SearchTextField.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SearchTextField.swift
+//  
+//
+//  Created by NamNH on 03/03/2022.
+//
+
+import SwiftUI
+
+import SwiftUI
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/3e6c3ff22fa5861f78566849d80733b6d3d53536/Packages/CommonUI/Sources/CommonUI/View/TextField/SearchTextField.swift#L10"></span>Imports should be unique.<br />
<code>duplicate_imports</code> <code>SearchTextField.swift:10</code></td>
    </tr>
  </tbody>
</table>